File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/Drive.java
package org.firstinspires.ftc.teamcode.commands;

import com.arcrobotics.ftclib.command.CommandBase;

import org.firstinspires.ftc.teamcode.core.MyRobot;
import org.firstinspires.ftc.teamcode.subsystems.DriveTrain;

public class Drive extends CommandBase {
	private DriveTrain dt;

	public Drive(DriveTrain dt_p, MyRobot robot_p) {
		dt = dt_p;
		addRequirements(dt);
	}

	@Override
	public void execute() {
		dt.rc_drive();
	}

	@Override
	public boolean isFinished() {
		return false;
	}
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/commands/Drive.kt
package org.firstinspires.ftc.teamcode.commands

import com.arcrobotics.ftclib.command.CommandBase
import org.firstinspires.ftc.teamcode.core.Robot
import org.firstinspires.ftc.teamcode.subsystems.MecanumDrive

class Drive(val dt: MecanumDrive, val robot: Robot) : CommandBase() {
    init { addRequirements(dt) }

    override fun execute() = dt.rcDrive()
    override fun isFinished(): Boolean = false
}